server/fileCompress/app.py

```diff
@@ -24,57 +24,69 @@ def lambda_handler(event, context):
     # Download files using reference data from DynamoDB
     files_json = response["Item"]["files"]
 
+    # Absolute path does not work in lambda, so instead defining relative path
+    tmp_dir_path = os.sep + "tmp"
+
     # Create zip file
-    zip_file_path = os.path.join("/tmp/", "temp_zip.zip")
+    zip_file_path = os.path.join(tmp_dir_path, "temp_zip.zip")
     temp_zip = zipfile.ZipFile(zip_file_path, "w")
 
     # Go through each file
     for file in files_json:
 
         # Get file path for s3 bucket and create file path to store in tmp
         file_ref_s3 = file["fileRef"]
-        file_ref_tmp = "/tmp/" + file["fileRef"].split("/")[-1]
+        file_ref_tmp = os.path.join(tmp_dir_path, os.path.split(file["fileRef"])[-1])
```

Comment:
You could use Path instead of os.path (Path is a replacement for os.path), as I've had issues with os.path when splitting, but if this works it's fine.